.travis.yml

```diff
@@ -199,8 +232,14 @@ script:
         cd ..
         USE_WHEEL_BUILD="--no-build"
     fi
-  - export SCIPY_AVAILABLE_MEM=3G
-  - python -u runtests.py -g -m $TESTMODE $COVERAGE $USE_WHEEL_BUILD -- -rfEX -n 3 2>&1 | tee runtests.log
+  - |
+    if [[ "$TRAVIS_CPU_ARCH" = "arm64" ]]; then
+      python -u runtests.py --parallel=4 | tee runtests.log
```

Comment:
Some of the options of the original command should be kept. Changing only what's necessary would be useful. Which is very little I think. The only difference needed seems to be adding `--parallel=4` and removing `n 3` (`-n` means `--numprocesses` to `pytest`).

`-g` does nothing, the other env vars are useful. Can't find the exact meaning for `2>&1`; `-rfEX` needs to be kept I believe.

Reply:
2>&1 redirects stderr to stdout

Reply:
`rfEX` is for pytest [output report](https://docs.pytest.org/en/latest/usage.html#detailed-summary-report)

> The -r options accepts a number of characters after it, with a used above meaning “all except passes”.
> 
> Here is the full list of available characters that can be used:
> 
>         f - failed
>         E - error
>         s - skipped
>         x - xfailed
>         X - xpassed
>         p - passed
>         P - passed with output
> 
>